capsul/engine/settings.py

```diff
@@ -273,10 +273,8 @@ def export_config_dict(self, environment=None):
                     env_conf = {}
                     for module in modules:
                         mod_conf = {}
-                        for config in session.configs(module, env):
-                            id = "%s-%s" % (config._id, env)
```

Comment:
Now, it is best to use f-string instead of `%` operator: `id = f"{condif._id}-{env}"`

Reply:
The modulo operator is very old and the f-string has been recommended since python 3.6.